util/test/checkChplInstall

```diff
@@ -153,20 +169,20 @@ TEST_EXEC_OUT=${TEST_JOB}.exec.out
 
     EXEC_OPTS="$(cat ${TEST_DIR}/${TEST_JOB}.execopts)"
 
-    echo "Running test job."
+    log_info "Running test job."
     ./${TEST_JOB} -nl${NUMLOCALES} ${EXEC_OPTS} | sort > ${TEST_EXEC_OUT} 2>&1
-    echo "Test job complete."
+    log_info "Test job complete."
 
     # Check result
     DIFF_OUTPUT=$(diff -q ${TEST_EXEC_OUT} ${GOOD})
 
     # Return success code (0) if diff is good
     if [ -z "${DIFF_OUTPUT}" ]; then
-    echo "Installation works as expected!"
+    log_info "Installation works as expected!"
```

Comment:
Might be nice to have a log_success that prints out `[Success]` to make it more obvious that things are in good shape